Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabStrip.css

```diff
@@ -17,16 +17,11 @@
   flex-grow: 99999;
```

Comment:
The changes are applied to the wrong context. TabStrip is used standaline and with TabbedMenu and TabbedMultiView. The changes should only apply to TabbedMenu.

We need to reset the unwanted changes in TabbedMenu while leaving the original for TabStrip and TabbedMultiView.

The first element is the TabbedMenu, the second and third elements are TabbedMultiView and TabStrip. Only the TabbedMenu should be changed in this task.
![image](https://user-images.githubusercontent.com/6972548/191954724-87014e2e-0bb7-4ed1-b36e-ee93fdebad08.png)